maltiverse/operations.py

```diff
@@ -0,0 +1,121 @@
+""" Copyright start
+  Copyright (C) 2008 - 2023 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end """
+import requests
+from connectors.core.connector import get_logger, ConnectorError
+from requests import exceptions as req_exceptions
+logger = get_logger('maltiverse')
+
+
+class Maltiverse(object):
+
+    def __init__(self, config):
+        self.server_url = config.get('server_url').strip()
+        self.api_key = config.get('api_key')
+        self.headers = {'accept': 'application/json', 'Authorization': 'Bearer {}'.format(self.api_key)}
+        if not self.server_url.startswith('https://'):
```

Comment:
please change the if condition to this
if not self.server_url.startswith('https://') and not self.server_url.startswith('http://'):